components/CheckDocumentation/vacuum/TxidWraparound.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+
+import {
+  CheckDocs,
+  CheckGuidanceProps,
+  CheckTriggerProps,
+} from "../../../util/checks";
+import { useSmartAnchor } from "../../SmartAnchor";
+
+const TxidWraparoundTrigger: React.FunctionComponent<CheckTriggerProps> = ({
+  config,
+}) => {
+  const warning = config.settings["warning_threshold_pct"] as number;
+  const critical = config.settings["critical_threshold_pct"] as number;
+  return (
+    <p>
+      Detects when the transaction ID utilization on the server is more than{" "}
+      <code>{warning}%</code> and creates an issue with severity "warning".
+      Escalates to "critical" once the utilization exceeds{" "}
+      <code>{critical}%</code>. Resolves once the utilization falls below the
+      threshold.
+    </p>
+  );
+};
+
+const TxidWraparoundGuidance: React.FunctionComponent<CheckGuidanceProps> = ({
+  urls: { backendsUrl },
+}) => {
+  const Link = useSmartAnchor();
+  return (
+    <div>
+      <h4>Impact</h4>
+      <p>
+        The transaction ID usage is high. Postgres usually runs anti-wraparound
+        autovacuums to keep the usage low by freezing old rows. However, if such
+        VACUUMs are either not running or are running but unable to freeze rows
+        for some reasons, the transaction usage keeps growing.
+      </p>
+      <p>
+        The anti-wraparound VACUUM needs to happen at some point and this is not
+        avoidable. If VACUUM is somehow not run in timely manner, the upcoming
+        VACUUM becomes more and more expensive, potentially causes the overall
+        performance degradation or simply takes really long time to finish. The
+        anti-wraparound VACUUM holds <code>SHARE UPDATE EXCLUSIVE</code> lock,
+        which can block DDL statements.
+      </p>
+      <p>
+        When the transaction ID usage reaches to 99.85% (3M transactions left),
```

Comment:
```suggestion
        If the transaction ID usage reaches to 99.85% (3M transactions left),
```

Hopefully no one actually has to deal with a "when" here :slightly_smiling_face:.